addons/web/controllers/webmanifest.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+import mimetypes
+
+from odoo import http
+from odoo.exceptions import AccessError
+from odoo.http import request
+from odoo.tools import ustr, file_open
+
+
+class WebManifest(http.Controller):
+
+    def _get_shortcuts(self):
+        module_names = ['mail', 'crm', 'project', 'note']
```

Comment:
the 'note' module is probably wrong here as the module has been removed recently https://github.com/odoo/odoo/pull/133101